docs/advanced-usage.md

```diff
@@ -31,6 +31,21 @@ fail or not.
 If you don't specify a `penalty` configuration, the plugin will use the [default threshold values][penaltyextensioncode], which are to
 allow any warning, but break the build on any error.
 
+It is also possible to use below convenience configuration to control failure thresholds
+
+```gradle
+staticAnalysis {
+    // no failure
+    penalty none
+    
+    // fail only on errors (default)
+    penalty failOnErrors
```

Comment:
Maybe it's nitpicking, but I'd rather split this a little bit, as people might be confused by the example. What about:
>
> It's also possible to use one of the default `penalty` profiles instead of specifying thresholds explicitly as follows:
<table>
<tr><td><pre>staticAnalysis {</br>&nbsp;&nbsp;penalty none</br>}</pre></td><td>In this case the build won't fail no matter how many violations (warnings or error) are found</><tr>
<tr><td><pre>staticAnalysis {</br>&nbsp;&nbsp;penalty failOnErrors</br>}</pre></td><td>Use this if you want to fail fast: any warning or this is the default policy and will break the build if any error is found. Warnings instead are logged but ignored during the evaluation.</><tr>
<tr><td><pre>staticAnalysis {</br>&nbsp;&nbsp;penalty failOnWarnings</br>}</pre></td><td>This is the default policy and will break the build if any warning or error is found.</><tr>
</table>

Reply:
I think it deserved a sub-section now. Updated. How does it look?